vulture/core.py

```diff
@@ -384,9 +389,20 @@ def visit_ClassDef(self, node):
 
     def visit_FunctionDef(self, node):
         for decorator in node.decorator_list:
-            if getattr(decorator, 'id', None) == 'property':
+            if isinstance(decorator, ast.Call):
+                decorator = decorator.func
+            if isinstance(decorator, ast.Attribute):
+                name = decorator.attr
+            else:
+                name = decorator.id
```

Comment:
This looks like we can't actually ignore "app.route", right? This contradicts the documentation. I think we should take the full decorator "name" into account, e.g., `f.foobar()` can be ignored with `f.foobar()` but not with `foobar`. I can imagine that we need a helper function for retrieving the decorator name from a decorator node, since these names can be nested more deeply, I guess, like `a.b.c.foo`.

Also, I think for ignoring decorators we should require "@" signs. So the documentation should talk about `--ignore-names "@app.route*"` (note the quotes which prevent bash from expanding the input).

Reply:
Actually, I think we should ignore the round brackets and everything in them. So `@app.route` ignores the Flask decorators.

Reply:
Yes, you are completely right. I don't how this slipped by. 😕. Let me make the changes.